eoscostarica/lifebank
# refactor(webapp): add email badge

Resolves #866 

### What does this PR do?
Add badge to donation email

### How should this be tested?
- Make run
- Make a donation
- Check there is a badge in donation email
